packages/@webex/internal-plugin-metrics/src/metrics.types.ts

```diff
@@ -67,10 +67,11 @@ export interface ClientEvent {
 }
 
 export interface BehavioralEvent {
-  // TODO: not implemented
-  name: 'host.meeting.participant.admitted' | 'sdk.media-flow.started';
-  payload?: never;
-  options?: never;
+  context: any;
+  name: string;
```

Comment:
IMO we should tighten this up

agent should be strongly typed (user, redux, system, etc)
product should be strongly typed (contactcenter, webclient, etc)
verb should be strongly typed (create, invite, schedule, etc)
target can be loose (string) although preferred if it's not (pmr, agent, etc)